scripts/generate-base-files-info-json.py

```
@@ -1,54 +1,347 @@
+        for line in package_entry.split('\n'):
+            if ':' in line:
+                key, value = line.split(':', 1)
+                package_info[key.strip().lower()] = value.strip()
+
+        # Check if this is our package
+        if package_info.get('package', '').lower() == package_name.lower():
+            return package_info
+
+    return None
+
+def synthesize_binary_package_filename(package_info, release_name):
```

Ruff (0.14.7)

Unused function argument: `release_name`

(ARG001)